app/jobs/spree/seo/file_dump/update_seo_job.rb

```diff
@@ -0,0 +1,53 @@
+module Spree
+  module Seo
+    module FileDump
+      class UpdateSeoJob < BaseSeoJob
+        class UpdateSeoJobError < StandardError; end
+
+        # @param filename [String]
+        # @param locale [String]
+        def perform(filename, locale, args)
+          @locale = locale
+          @args  = args
+          envs.each do |env|
+            path = "#{default_path}/#{env}/#{filename}"
+            return unless load_file(path) && seo_key_exists
+
+            @seo_index = find_existing_key
+            if @seo_index.nil?
+              add_new_seo
```

Comment:
To są zadania z sidekiq, wolał bym abyśmy przekazywali parametry do metod zamiast ustawiać zmiennej obiektu. 
```suggestion
              add_new_seo(locale, args)
```